### DISCUSSION TITLE
Status/roadmap for Deno test coverage support?

### DISCUSSION THREAD

**Discussion by rivy** [denoland/deno#9819] [Q&A] [answered]
I'm getting some odd coverage results (see #9865).

I think it's because coverage is still immature and undergoing development, but I can't find any docs or a roadmap as to what it does now and what the future is expected to be... Are either (or both) published somewhere?

**Comment by rivy** [comment]
@kitsonk , this is a current bug in the code coverage. I'm not sure why you moved it into a "Discussion".

**Reply by kitsonk** [reply]
> Status/roadmap for Deno test coverage support?

You were asking for a status/roadmap.

**Reply by rivy** [reply]
Really only peripherally ... Maybe I was being too conversational or verbose.

I've re-edited this as a request for status/roadmap and re-posted the coverage issue as an actual bug report.

**Comment by rivy** [comment, marked as answer]
Note the umbrella issue (https://github.com/denoland/deno/issues/12826) by @bartlomieju.
